doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2358,7 +2374,7 @@ NCSDK-18263: |NCS| samples may fail to boot on Thingy:53
 Bluetooth samples
 =================
 
-.. rst-class:: v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0
```

Comment:
This was fixed by @maje-emb in the following PR:

https://github.com/nrfconnect/sdk-nrf/pull/15960

NCS 2.7.0, 2.8.0 and 2.9.0 should be removed.

@maje-emb, it would be good to get confirmation from you

Reply:
@maje-emb is not available today, but I confirm this is fixed - `2.9.0` to be removed.

Reply:
As per above comment, I have only removed the 2.9.0 version.